src/anim/evaluator/anim-events.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * @class
+ * @name AnimEvents
+ * @classdesc AnimEvents stores a sorted array of animation events which should fire sequentially during the playback of an pc.AnimTrack.
+ * @description Create a new collection of animation events.
+ * - @param {object[]}  events- An array of animation events.
+ * @example
+ * const events = new pc.AnimEvents([
+ *     {
+ *         name: 'my_event',
+ *         time: 1.3, // given in seconds
```

Comment:
tiny thing, but list time as first property, as that is required .. all the rest is optional I think, including the name. Make it clearer somehow maybe.

Reply:
The name isn't optional as it's used to fire the event. I can make this clearer though.